packages/manager/modules/pci/src/projects/project/storages/containers/add/add.html

```diff
@@ -424,13 +439,13 @@ <h2 class="oui-header__description mb-5">
         data-cancel-text="{{:: 'pci_projects_project_storages_containers_add_cancel_label' | translate }}"
         data-submit-text="{{:: 'pci_projects_project_storages_containers_add_submit_label' | translate }}"
         name="storage_container_add_name"
-        data-valid="!$ctrl.isDiscoveryProject"
+        data-valid="!$ctrl.isDiscoveryProject && !$ctrl.isAddingRegion && !$ctrl.isAddingRegionError"
         data-prevent-next
     >
         <oui-field
             data-size="xl"
             data-help-text="{{ ::'pci_projects_project_storages_containers_add_pattern_help_storage-s3' | translate }}"
-            data-ng-if="$ctrl.isRightOffer() ? !$ctrl.archive && $ctrl.currentStep > 4 : !$ctrl.archive"
+            data-ng-if="$ctrl.isRightOffer() && !$ctrl.isLocalZone() ? !$ctrl.archive && $ctrl.currentStep > 4 : !$ctrl.archive && !$ctrl.isAddingRegion && !$ctrl.isAddingRegionError"
```

Comment:
maybe give a named function for this one since line is not easy to read and understand quickly here